test/expect-tests/fiber/fiber_tests.ml

```diff
@@ -871,35 +875,6 @@ let%expect_test "raise exception" =
        (fun () -> Pool.stop pool));
   [%expect {| Caught Exit |}]
 
-let%expect_test "execution context preservation in iter callback" =
```

Comment:
Just a note for whoever else might wonder why this test got removed: it no longer makes sense because the `iter` argument of `Fiber.run` accepts a pure callback, and `get` now lives in the `Fiber` monad.